app/helpers/comments_helper.rb

```diff
@@ -31,6 +34,22 @@ def should_be_hidden?(comment, root_comment)
     comment.hidden_by_commentable_user && comment != root_comment
   end
 
+  def high_number_of_comments?(comments_number)
+    comments_number > MAX_NBR_OF_COMMENTS_TO_RENDER
+  end
+
+  def view_all_comments?(comments_number)
+    comments_number > MIN_NBR_OF_COMMENTS_TO_RENDER
+  end
+
+  def number_of_comments_to_render
+    MAX_NBR_OF_COMMENTS_TO_RENDER
+  end
+
+  def comments_to_render(view)
+    view == "comments" ? MAX_NBR_OF_COMMENTS_TO_RENDER : MIN_NBR_OF_COMMENTS_TO_RENDER
+  end
+
```

Comment:
**suggestion (non-blocking):** what if we changed the name of the `comments_to_render` method? When I see that, I expect it to return `Comment` objects, but it actually returns an integer.

I was going to suggest `number_of_comments_to_render` as a name, but I see that is also being used. That said, it looks like that method is returning the constant so we can refer to the constant itself instead of using a method name, or change both method names.

Reply:
Hi @atsmith813 , I see your point. The thing is, there is 2 cases where you display comments(based on the 2 Constants), so I did not find a way to name 2 "number_of_comments".

Maybe I could use for the last(`comments_to_render`) this: `comment_count`. It implies correctly a return of a numeric, and in the controller, I use this very variable as a return for the `comments_to_render` method. Would that be easier to read?

Regarding the Constant, I prefer to use an accessor rather than to have Helper::CONSTANT in the view.

Reply:
Great idea @cyrillefr!